app/LoginPage/page.js

```diff
@@ -0,0 +1,82 @@
+"use client";
+import React, { useState } from "react";
+import { FiShoppingCart } from "react-icons/fi";
+import { FcGoogle } from "react-icons/fc";
+import { createUserWithEmailAndPassword, signInWithPopup } from "firebase/auth";
+import { auth, provider } from "../../FirebaseConfig/Firebase";
+
+function Login() {
+  const [email, setEmail] = useState("");
+  const [password, setPassword] = useState("");
+  // REDIRECT USER AFTER SIGNIN
+  const RedirectUser = (e) => {
+    window.location.replace("ProductPage");
+  };
+
+  // LOGIN USER WITH EMAIL AND PASSWORD
+  const LoginWithEmail = async (e) => {
+    try {
+      e.preventDefault();
```

Comment:
[Question]
Why do you need this?

Reply:
i added it when i was working on the Firebase authentication...so the botton will not refresh the page sir